src/Sylius/Component/Order/spec/Model/OrderItemSpec.php

```diff
@@ -26,14 +30,14 @@ function it_is_initializable()
         $this->shouldHaveType('Sylius\Component\Order\Model\OrderItem');
     }
 
-    function it_implements_Sylius_order_item_interface()
+    function it_implements_sylius_order_item_interface()
```

Comment:
Should be reverted. We unofficialy set up convention to write proper names from capital letter,